CAP/gap/CategoryMorphisms.gd

```diff
@@ -413,14 +413,21 @@ DeclareOperation( "AddIsAutomorphism",
 ###################################
 ##
 #! @Section Adding Morphisms to a Category
+#! @SectionLabel Adding_Morphisms_to_a_Category
 ##
 ###################################
 
+#! @Description
+#!  Adds <A>morphism</A> as a morphism to <A>category</A>.
+#! @Arguments category, morphism
 DeclareOperation( "Add",
                   [ IsCapCategory, IsCapCategoryMorphism ] );
 
+#! @Description
+#!  Adds <A>morphism</A> as a morphism to <A>category</A>.
+#! @Arguments category, morphism
```

Comment:
Should those two docstrings may be grouped, and then describe the difference explicitly, i.e., in one case, the morphism needs to be a `CapCategoryMorphism`?

Reply:
If we group the docstrings, only the filters of the first operation are visible in the manual, right? I don't think, this makes things clearer for the user. But I agree that we should make the relation between the two more clear. Thus, I propose the following:
```suggestion
#! @Description
#!  Adds <A>morphism</A> as a morphism to <A>category</A>. If <A>morphism</A> already lies in the filter <C>IsCapCategoryMorphism<C>, the operation <C>Add</C> can be used instead.
#! @Arguments category, morphism
```

Reply:
Very good!